yb-voyager/cmd/importData.go

```diff
@@ -948,26 +940,18 @@ func getInProgressFilePath(task *SplitFileImportTask) string {
 	path := task.SplitFilePath
 	base := filepath.Base(path)
 	dir := filepath.Dir(path)
-	parts := strings.Split(base, ".")
 
-	if len(parts) > 5 { //case when filename has schema also
-		return fmt.Sprintf("%s/%s.%s.%s.%s.%s.P", dir, parts[0], parts[1], parts[2], parts[3], parts[4])
-	} else {
-		return fmt.Sprintf("%s/%s.%s.%s.%s.P", dir, parts[0], parts[1], parts[2], parts[3])
-	}
+	base = base[0:len(base)-1] + "P" // *.C -> *.P
+	return fmt.Sprintf("%s/%s", dir, base)
```

Comment:
Nit: You don't need to split the `path` into `dir` and `base` to implement this logic. Just following should be enough:
```
path := task.SplitFilePath
return path[0:len(path)-1] + "P"
```

Same applies for the `getDoneFilePath()` as well.

Reply:
done